pallets/stableswap/src/lib.rs

```diff
@@ -517,7 +535,7 @@
 		/// - 'min_amount_out': minimum amount to receive
 		///
 		/// Emits `LiquidityRemoved` event when successful.
-		#[pallet::call_index(4)]
+		#[pallet::call_index(5)]
```

Comment:
it is not issue as pallet is not deployed yet anywhere, but this would be breaking change

Reply:
as long as 5 didn't exist before and 4 is not replaced by a different extrinsic it at least doesn't require a transaction version bump